user_guide/accounts.rst

```diff
@@ -0,0 +1,287 @@
+.. _user-guide-accounts:
+
+Accounts
+========
+
+Account records ("accounts") represent  represent a person, company or group of people you do business activities with. 
+An account can aggregate details of all the :term:`customer identities <Customer Identity>` assigned to it, providing for 
+a 360-degree view of the account activity. 
+
+From this article you will learn how to create new account, understand 
+their :ref:`View page <user-guide-ui-components-view-pages>`, manage and merge existing accounts and analyse their  
+details with OroCRM reports.
+
+Accounts and Customers
+----------------------
+
+If a :ref:`Magento channel <user-guide-magento-channel>` has been created in the system, a new account is created for
```

Comment:
Lets start account story with the case when no channels are configured and explain channels impact down the road